STM32/DC/Core/Src/DCBoard.c

```diff
@@ -191,7 +272,7 @@ static void actuatePins(ActuationInfo actuationInfo)
         return;
     }
 
-    if (actuationInfo.timeOn == -1 && (actuationInfo.percent == 100 || actuationInfo.percent == 0))
+    if (actuationInfo.timeOn == -1000 && (actuationInfo.percent == 100 || actuationInfo.percent == 0))
```

Comment:
I'm guessing the update from -1 to -1000 has to do with the bug mentioned in the PR summary. Can you elaborate on the difference here?

Reply:
Just noticed the same bug is present in the AC board